### DISCUSSION TITLE
GeometryToTexture and Normals

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#477] [Answers & Questions] [answered]
i'm messing with geometryToTexture and struggle with normals. The node has a normals setting but the output (visually) looks the same as VertexPos. And then i don't know where to feed this into the final geometry.

I also tried calculateNormals but its doesnt do anything in my case. At least not what i expect.

What i'm trying to do is manipulating a sphere by a texture (imagecompose) and have it displayed as a mesh with a material...and not just a pointcloud.

see demo here https://cables.gl/p/63d902869755ffa26886b0d0?s=rvJZvnVSJWEfONzs

**Comment by pandrr** [comment, marked as answer]
please try again on dev.cables.gl it received some bugfixes and tweaks there
